apps/web/app/(authenticated)/(app)/app/apis/[apiId]/settings/update-api-name.tsx

```diff
@@ -53,9 +53,14 @@ export const UpdateApiName: React.FC<Props> = ({ api }) => {
     },
   });
   async function onSubmit(values: z.infer<typeof formSchema>) {
+    if (!values.name) {
+      return toast.error("Please provide a valid name before saving.");
+    }
     if (values.name !== api.name) {
```

Comment:
shouldn't this be the inverse?

if they're the same, we toast

Reply:
Should I add the condition something like this.

```
if (values.name !== api.name || !values.name) {
   return toast.error("Please provide a valid name before saving.");
}
```

Reply:
No, if `values.name !== api.name` is true, then the new and old name are different, why are we toasting an error in this case?

Reply:
Ohh, my bed.